proto/viam/app/v1/app.proto

```diff
@@ -447,6 +449,14 @@ message UpdateBillingServiceRequest {
 
 message UpdateBillingServiceResponse {}
 
+message GetBillingServiceRequest {
+  string org_id = 1;
+}
+
+message GetBillingServiceResponse {
+  BillingAddress billing_address = 1;
+}
```

Comment:
@RoxyFarhad what other fields are you picturing here? you listed a few in the ticket description but wanted to know what you included in the email

Reply:
copied from the email:
```
message GetBillingServiceConfigResponse {
  BillingAddress billing_address = 1; 
  string support_email = 2;
  string logo_url = 3;
  string billing_dashboard_url = 4; 
}
```

Reply:
updated